### DISCUSSION TITLE
Strange Behavior in TW 5.2.3p

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#6785] [Q&A] [answered]
I am working on https://tiddlywiki.com/prerelease/

I created a global macro in myMacros tiddler like below

```
\define mc() color is red
```

and save

then in a Test tiddler I called ma like 

```
<<mc>>
```

It works as expected. 

NOW, edit the myMacros tiddler, while you are in edit mode all changes propagate, this is strange and one needs to save myMacros tiddler and then see the effects. For example while you are in edit mode change `\define mc() color is red` to `\define mc() color is blue`  without saving, you see in Test tiddler <<mc>> results in `color is blue`

----

Second issue

If a tiddler tagged with $:/tags/Macro its macro are global and accessible from all other tiddlers
If you add field: code-body set to yes, the macros will not work and are not in access

**Comment by pmario** [comment]
hmm. That's strange -- I can't replicated that. Which OS and Browser?   I'm testing with Windows 11 pro and FF latest atm

**Comment by pmario** [comment, marked as answer]
@kookma ... Please don't use Discussions for issues. If issues are closed, they are archived in a different list. Discussions will always be visible.

**Reply by pmario** [reply]
@Jermolene ... could you move this one to issues?

**Reply by kookma** [reply]
Please ignore my question! The issue was on my side!
I cannot reproduce it online anymore! TW works fine
Sorry!

**Comment by kookma** [comment]
@pmario
It seems something strange was in my side.  There is no propagation until I save the `myMacros` tiddler.
Sorry for inconvenience.

Browser: Edge 103
OS: Windows 10
TW: 5.2.3p
